packages/office-addin-lint/src/lint.ts

```diff
@@ -11,8 +12,12 @@ const prettierFilePath = path.resolve(prettierPath, "../bin-prettier.js");
 const esLintConfigPath = path.resolve(__dirname, "../config/.eslintrc.json");
 
 function execCommand(command: string) {
-  const execSync = require("child_process").execSync;
-  const child = execSync(command, { stdio: "inherit" });
+  try {
+    const execSync = require("child_process").execSync;
+    const child = execSync(command, { stdio: "inherit" });
+  } catch (err) {
+    throw err;
```

Comment:
It doesn't make sense to `catch err` just to `throw err`. It is the same as no catch at all. Perhaps you removed telemetry reporting here but missed removing the catch as well.